### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by chuq2b2**
Create the layout of the project: NavBar (including the SearchBar) and SideBar.

### CODE REVIEW

**Review by hphng — COMMENTED** [review]
hay

File: components/base/Header/Bars.js
Comment (Minhtud04) [inline components/base/Header/Bars.js:1]: uay hay ghe day t di

**Review by hphng — APPROVED** [review]
hay

**Review by Minhtud04 — APPROVED** [review]
this is Minh

**Pull request merged**